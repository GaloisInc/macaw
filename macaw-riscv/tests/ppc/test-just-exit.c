#include "util.h"

void _start() {
  EXIT();
}
