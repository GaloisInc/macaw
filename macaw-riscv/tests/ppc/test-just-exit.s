	.file	"test-just-exit.c"
	.section	".text"
	.align 2
	.globl _start
	.section	".opd","aw"
	.align 3
_start:
	.quad	.L._start,.TOC.@tocbase,0
	.previous
	.type	_start, @function
.L._start:
	std 31,-8(1)
	stdu 1,-64(1)
	mr 31,1
#APP
 # 4 "test-just-exit.c" 1
	li 0,1
sc
 # 0 "" 2
#NO_APP
	nop
	addi 1,31,64
	ld 31,-8(1)
	blr
	.long 0
	.byte 0,0,0,0,128,1,0,1
	.size	_start,.-.L._start
	.ident	"GCC: (Ubuntu 7.2.0-6ubuntu1) 7.2.0"
