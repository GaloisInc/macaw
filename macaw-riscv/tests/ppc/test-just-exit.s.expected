R { funcs = [(0x10000148, [(0x10000148, 20)])]
  , ignoreBlocks = [0x1000015c]
  }